src/backend/common/helpers/opr_helper.py

```diff
@@ -0,0 +1,126 @@
+# taken from https://github.com/the-blue-alliance/the-blue-alliance/blob/bd16a7daf19774a2dc84509f71fa786c45d0a99e/old_py2/helpers/matchstats_helper.py

```

Comment:
I forgot to make sure git knows where this is moved from, oops. I can fix that later